tests/unit/versioning/VersionID.js

```diff
@@ -9,83 +10,121 @@ function randkey(length) {
     return key;
 }
 
-describe('test generating versionIds', () => {
-    const count = 1000;
+process.env.VID_CRYPTO_PASSWORD = randkey(64);
+
+function generateRandomVIDs(count) {
     const vids = Array(count).fill(null);
     for (let i = 0; i < count; i++) {
         vids[i] = VID.generateVersionId(randkey(15), 'PARIS');
     }
-    process.env.VID_CRYPTO_PASSWORD = randkey(64);
+    return vids;
+}
+
+const count = 1000000;
+
+describe('test generating versionIds', () => {
+    describe('legaxy hex encoding', () => {
+        env.S3_VERSION_ID_ENCODING_TYPE = 'hex';
+        const vids = generateRandomVIDs(count);
 
-    it('sorted in reversed chronological and alphabetical order', () => {
-        for (let i = 0; i < count; i++) {
-            if (i !== 0) {
+        it('sorted in reversed chronological and alphabetical order', () => {
+            for (let i = 1; i < count; i++) {
                 assert(vids[i - 1] > vids[i],
                     'previous VersionID is higher than its next');
             }
-        }
-    });
+        });
 
-    // nodejs 10 no longer returns error for non-hex string versionIds
-    it.skip('should return error decoding non-hex string versionIds', () => {
-        const encoded = vids.map(vid => VID.hexEncode(vid));
-        const decoded = encoded.map(vid => VID.hexDecode(`${vid}foo`));
-        decoded.forEach(result => assert(result instanceof Error));
-    });
+        // nodejs 10 no longer returns error for non-hex string versionIds
+        it.skip('should return error decoding non-hex string versionIds', () => {
```

Comment:
Let's remove it if it's no longer applicable, in a separate commit if we really want to keep a trace of the removal.